addons/web/static/tests/webclient/commands/command_service_tests.js

```diff
@@ -407,19 +403,17 @@ QUnit.test("display shortcuts correctly for MacOS ", async (assert) => {
     // Register some commands
```

Comment:
You should also add a test asserting that the command palette takes into account the overlayModifier of the hotkeyService, even if it has been replaced.